pkg/vdr/panacea_vdr.go

```diff
@@ -44,6 +45,23 @@ func (r *PanaceaVDR) Resolve(didID string, _ ...vdr.DIDMethodOption) (*did.DocRe
 		return nil, fmt.Errorf("failed to parse DID document: %w", err)
 	}
 
+	var vms []did.VerificationMethod
+	for _, vm := range doc.VerificationMethod {
+		var verificationMethod did.VerificationMethod
+		verificationMethod = vm
+		if btcec.IsCompressedPubKey(vm.Value) {
+			pubKey, err := btcec.ParsePubKey(vm.Value, btcec.S256())
+			if err != nil {
+				return nil, fmt.Errorf("invalid secp256k1 public key: %w", err)
+			}
+
+			verificationMethod.Value = pubKey.SerializeUncompressed()
+		}
+		vms = append(vms, verificationMethod)
+	}
+
+	doc.VerificationMethod = vms
```

Comment:
As we discussed in Slack, currently the public key of DID from Panacea is compressed version, but aries use uncompressed version of public key for proof verification.
Thus I added this converting logic.